inf-elixir.el

```diff
@@ -81,6 +81,12 @@ printed instead."
   :type 'buffer
   :group 'inf-elixir)
 
+(defcustom inf-elixir-on-send-switch-to-repl t
```

Comment:
Not a requirement, but the name is a bit awkward.  What do you think about `inf-elixir-switch-to-repl-on-send`?

Reply:
@J3RN thank you for the comment. Adjusted accordingly